### DISCUSSION TITLE
Using old value in update method

### DISCUSSION THREAD

**Discussion by Ripul01** [prisma/prisma#18242] [Q&A]
Can we use the previous value stored in the database to update many documents? Here are the details.

I am working on an accounting app and want to allow the user to change the order no format. Suppose the order numbers saved are: "JOUR-22/23-1",  "JOUR-22/23-2", "JOUR-22/23-3" and so on.

Now somewhere down the line, the user decides the order no. format should be JOR instead of JOUR. Can I update all the previous records to only replace JOUR with JOR?

The only way I've found to solve this is to fetch all the records with their order no's and make individual update calls in a for loop. This, however, takes a long time and makes a lot of requests to the database when there are thousands of records.

Is there a better way to solve this problem?

**Comment by nurul3101** [comment]
Hello @Ripul01 👋 

Instead of using for loop, you can use a replace function with a [raw SQL query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).

It could look something like this:
```sql
UPDATE orders
SET order_number = REPLACE(order_number, 'JOUR', 'JOR')
WHERE order_number LIKE 'JOUR%'
```

**Reply by nurul3101** [reply]
@Ripul01 If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
